lib/colors.ts

```diff
@@ -14,41 +13,26 @@ import {
   modeLab,
   modeP3,
   parse as originParse
-  // @ts-expect-error
 } from 'culori/fn'
+import type { Color, Rgb as RgbColor, Lch } from 'culori/fn'
```

Comment:
```suggestion
import type { Color, Rgb as RgbColor, Lch as LchColor } from 'culori/fn'
```

For the consistency

Reply:
And can you move `import type` above normal `import`?

Reply:
https://github.com/evilmartians/oklch-picker/pull/35/commits/fe8910609912c9bbd93bf4d5619bd2bbc92e4e3e